cypress/e2e/circulation-log/user-without-barcode.cy.js

```diff
@@ -0,0 +1,119 @@
+import uuid from 'uuid';
+import { getTestEntityValue } from '../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import UserEdit from '../../support/fragments/users/userEdit';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import { Locations } from '../../support/fragments/settings/tenant/location-setup';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import PatronGroups from '../../support/fragments/settings/users/patronGroups';
+import checkInActions from '../../support/fragments/check-in-actions/checkInActions';
+import searchPane from '../../support/fragments/circulation-log/searchPane';
```

Comment:
change your imports to Uppercase please, (searchPane -> SearchPane, checkInActions -> CheckInActions)